recipes/nng/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
Right, this is just the CMakeLists of the test package.

What got me confused is that nng requires CMake 3.13 in its own CMakeLists (which it seems we have to run, as @madebr pointed out). Does this need to be reflected anywhere?